jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/definition/details/ProcessDefinitionDetailsPresenter.java

```diff
@@ -160,7 +163,8 @@ public IsWidget getDetailsView() {
     }
 
     public IsWidget getProcessDiagramView() {
-        return processDiagramPresenter.getView();
+        processDefinitionDiagramView.setDiagram(processDiagramPresenter.getView());
```

Comment:
@nmirasch do you think this view could be moved to the presenter, this looks a bit strange to me. Maybe refactor a bit the common bits between definition and instance diagrams if needed.

Reply:
oks